lib/auth_browser.dart

```diff
@@ -260,7 +260,7 @@ class HybridFlowResult {
 class _AutoRefreshingBrowserClient extends AutoRefreshDelegatingClient {
   AccessCredentials credentials;
   ImplicitFlow _flow;
-  Client _authClient;
+  late Client _authClient;
```

Comment:
Can we move the initializing expression here?


```dart
late _authClient = authenticatedClient(baseClient, credentials);
```

Reply:
It can't go inline but it can go in an initializer; it looks like that doesn't work because `baseClient` is from `super`, but actually `baseClient` is just exactly `client`. Done.